api-report/container-runtime.api.md

```diff
@@ -74,6 +74,8 @@ export enum ContainerMessageType {
     Rejoin = "rejoin"
 }
 
+// Warning: (ae-forgotten-export) The symbol "IGarbageCollectionRuntime" needs to be exported by the entry point index.d.ts
```

Comment:
Can we ignore this warning?

Reply:
Good catch. Let me try to find out.

Reply:
@markfields Do you have any idea how we can avoid this?

Reply:
@tylerbutler @anthony-murphy Do you guys have any guidance or opinion on this? Basically, ContainerRuntime implements an interface (IGarbageCollectionRuntime) that is not public.

Reply:
Seems ok to revisit later.  I can imagine an alternative where we don't say it implements it in the code but we have a type test that proves that you can call a ContainerRuntime by the interface type and TypeScript accepts it.  Less helpful when developing though.

Reply:
@Josmithr DO you know if api-extractor provides any options that would be helpful here?

Reply:
Not to my knowledge, except by disabling the ae-forgotten-export rule altogether. But I think the warning is an important one. Even if we don't expect our consumers to use the interface directly, the fact that one of our exported classes implements that interface means that it implements some semantic some semantics that we think are meaningful, so we really shouldn't hide them away. 

IMO if we see warnings like this and don't think the interface should be exported, then it's a sign that we want to rethink the type hierarchy in some way.

Also with the way TSDoc works, if we remove the interface from the package exports, then we will lose any documentation the classes inherit from the interface.